### ISSUE TITLE
Setting page numbers in Oultine form doesn't transfer to Text form

### ISSUE THREAD

**Issue by aharonium** [opensiddur/opensiddur-client#59]
It would be great if the page numbers set in the outline transferred to the page numbers in the text input form.

**Comment by EfraimFeinstein** [comment]
Which text input form?

**Comment by aharonium** [comment]
Sorry. The form called "Texts" in between "Outlines" and "Annotations"

**Comment by EfraimFeinstein** [comment]
The page numbers are set correctly at the lowest level of the outline, but
possibly not at the higher levels. Will have to check that out.

**Comment by EfraimFeinstein** [comment]
I tried to reproduce this bug and I can't. Can you show the outline and outline item (resource) where this happened?

**Comment by aharonium** [comment]
Yeah, I can't reproduce it either. This makes me think that it was unexpected behavior from after I saved the outline but before I "executed" it. Why are there two steps (save and execute)?  Should these two steps be integrated? I'm sure there's probably a reason why they aren't or can't be yet.

**Comment by aharonium** [comment]
Here's one thing I am seeing. I can't correct the association between the page numbers and the transcription in the Texts form page regardless of where I change the page number (or page range).

**Comment by EfraimFeinstein** [comment]
Try going to the Texts view, open the file and press Metadata > Set sources
and status

The pages are editable there.

Save the file if the new range is correct.
